roles/postgres-infra/templates/postgres-cluster.yaml.j2

```diff
@@ -0,0 +1,89 @@
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+metadata:
+  name: ripsaw-postgres-cluster
+  namespace: {{ operator_namespace }}
+spec:
+  init_containers:
+  - name: date
+    image: busybox
+    command: [ "/bin/date" ]
+  teamId: "RIPSAW"
+  volume:
+    size: {{ postgres.storage.volume_size }}
+    storageClass: {{ postgres.storage.class_name }}
+  numberOfInstances: {{ postgres.servers.size }}
+  users: #Application/Robot users
+    zalando:
+    - superuser
+    - createdb
+  enableMasterLoadBalancer: true
+  enableReplicaLoadBalancer: true
+  allowedSourceRanges: # load balancers' source ranges for both master and replica services
+  - 127.0.0.1/32
+  databases:
+    foo: zalando
+#Expert section
```

Comment:
Do we want to expose any of this in the CR to allow users to overwrite these values?

Reply:
Most likely, yes. My initial aim was to enable automation for CI validation of pgbench, mostly, so defaults are fine. I might need feedback from a postgres expert to best determine what to expose in the CR.